pkg/controller/machine/controller.go

```diff
@@ -344,6 +346,10 @@ func (r *ReconcileMachine) drainNode(machine *machinev1.Machine) error {
 		DryRun: false,
 	}
 
+	if nodeIsUnreachable(node) {
+		drainer.SkipWaitForDeleteTimeoutSeconds = skipWaitForDeleteTimeoutSeconds
```

Comment:
Can we include a logging message here e.g "This node is unreachable, draining will wait for pod deletion during `skipWaitForDeleteTimeoutSeconds` after the request and will skip otherwise."

Reply:
Makes sense, can we do it after master opens?

Reply:
@enxebre Why not skip the drain entirely we know the node to be unreachable?

Reply:
The idea is that kubelet unreachability might be temporary. The intention is to tolerate unreachability during a reasonable timeframe i.e 5 min before considering the node dead and deleting the underlying infra, therefore disrupting app intent for graceful shutdowns and PDB policies.